core/src/main/scripts/importer/validateData.py

```diff
@@ -1896,6 +1896,11 @@ def checkLine(self, data):
                                'cause': value})
             elif col_name in ('loc.start', 'loc.end'):
                 try:
+                    # convert possible scientific notation to python scientific notation
+                    if "e+" in value:
+                        value = float(value.replace("e+", "e"))
+                        if int(value) == value:
+                            pass
```

Comment:
This conditional does not seem to do anything in either case. Could be useful raise a ValueError or log something if [value.is_integer()](https://docs.python.org/2/library/stdtypes.html#float.is_integer) returns false?